### PR TITLE
Clarify that changefeed jobs will not resume after full cluster restore

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-8287

We now explicitly do not resume changefeed jobs after a full-cluster restore. This PR adds notes to the restore and changefeed docs to that effect.

### CODE REVIEW

File: src/current/v23.1/restore.md [hunk 1]
```diff
@@ -170,10 +170,11 @@ You can exclude a table's row data from a backup using the [`exclude_data_from_b
...
 - All [views](views.html)
 
-Also, a full cluster restore will:
+Also, consider that:
 
-- Restore [temporary tables](temporary-tables.html) to their original database during a full cluster restore.
-- Drop the cluster's `defaultdb` and `postgres` [pre-loaded databases](show-databases.html#preloaded-databases) before the restore begins. You can only restore `defaultdb` and `postgres` if they are present in the original [backup](take-full-and-incremental-backups.html).
+- [Temporary tables](temporary-tables.html) will be restored to their original database during a full cluster restore.
+- The restore will drop the cluster's `defaultdb` and `postgres` [pre-loaded databases](show-databases.html#preloaded-databases) before the restore begins. You can only restore `defaultdb` and `postgres` if they are present in the original [backup](take-full-and-incremental-backups.html).
+- [Changefeed jobs](change-data-capture-overview.html) will not resume automatically on the new cluster. It is necessary to manually start changefeeds after a full cluster restore.
```
Comment (kathancox) [inline src/current/v23.1/restore.md:177]: @miretskiy Should this actually be "manually create changefeeds after a full cluster restore"?

Comment (miretskiy) [reply src/current/v23.1/restore.md:177]: Yes

**Review by miretskiy — APPROVED** [review]

File: src/current/v23.1/restore.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ For details on restoring across versions of CockroachDB, see [Restoring Backups
 ## Considerations
 
 - `RESTORE` cannot restore backups made by newer versions of CockroachDB.
-- `RESTORE` only supports backups taken on a cluster on a specific major version into a cluster that is on the same version or the next major version. 
+- `RESTORE` only supports backups taken on a cluster on a specific major version into a cluster that is on the same version or the next major version.
```
Comment (ashleykwtam) [inline src/current/v23.1/restore.md:25]: can link out to https://www.cockroachlabs.com/docs/stable/restoring-backups-across-versions.html

**Review by ashleykwtam — APPROVED** [review]
LGTM!

File: src/current/v23.1/restore.md (hunk 2 above)
Comment (kathancox) [reply src/current/v23.1/restore.md]: Sure, I can add this link as an opportunistic edit not related to the PR.

**Pull request merged**